vassal-app/src/main/java/VASSAL/build/module/PlayerRoster.java

```diff
@@ -622,13 +622,36 @@ protected boolean allSidesAllocated() {
 
   /**
    * @param side Name of a side to see if it's a "solo side"
-   * @return True if the side is "Solitaire", "Solo", "Moderator", or "Referee"
+   * @return True if the side is "Solo" or begins "Solitaire", "Solo:", "Moderator", "Referee" or a Side
+   * found in the Global Translatable Message Property: VassalSkipSideList (a comma-separated list).
+   * To test specifically for Referee/Moderator, use isRefereeSide()
    */
   public static boolean isSoloSide(String side) {
-    return Resources.getString("PlayerRoster.solitaire").equals(side) ||
-           Resources.getString("PlayerRoster.solo").equals(side) ||
-           Resources.getString("PlayerRoster.moderator").equals(side) ||
-           Resources.getString("PlayerRoster.referee").equals(side);
+    // FIXME: A future version of Vassal may provide a Side Type attribute in Definition of Sides that would allow this function to be rationalised.
+    // See also isRefereeSide()
+    return side.startsWith(Resources.getString("PlayerRoster.solitaire")) ||
+            side.equals(Resources.getString("PlayerRoster.solo")) ||
+            side.startsWith(Resources.getString("PlayerRoster.solo") + ":") ||
+            side.startsWith(Resources.getString("PlayerRoster.moderator")) ||
+            side.startsWith(Resources.getString("PlayerRoster.referee")) ||
+            side.startsWith(Resources.getString("PlayerRoster.auxiliarySidePrefix")) || // auxiliary side prefix
+            side.startsWith(SOLITAIRE) || // these checks needed in case a translated module has not localised these side names
```

Comment:
Are these really needed? If a translated module hasn't localized the side names don't they come through in English anyway?

Reply:
I think the actual problem is in the ChessClock which calling isSoloSide with the translated Sides, it should be checking untranslated sides, but displaying translated sides,

The concern is that ChessClocks now have translated sides recorded in them?, they should be untranslated sides.

Reply:
@BrentEaston So you think this usage is okay? Also are you okay with the various changes to "startsWith"?

Reply:
@BrentEaston And we're comparing both Resources.getString("Some String") and the literal "Some String" in here. Surely only one is right? Or is one way being left in because ChessClock is broken? (are other things broken too?)